### PR TITLE
fix: replace social links

### LINKED ISSUE #311: change the Twitter handle to the 4c community one
### Is your feature request related to a problem? Please describe.

change the current Twitter handle redirection to the 4c Twitter account:

https://twitter.com/4ccommunityhq

you should update the docusaurus.config.js file

line 105
image

image

### Describe the solution you'd like

clicking on the Twitter link redirects you to https://twitter.com/4ccommunityhq

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by RakeshSangem**
## 🛠️ Fixes Issue

Closes #311  

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

### CODE REVIEW

**Comment by FrancescoXX** [comment]
@ardasevinc can you check and merge those?

**Comment by ardasevinc** [comment]
> @ardasevinc can you check and merge those?

on it

File: docusaurus.config.js [hunk 1]
```diff
@@ -98,11 +98,11 @@ const config = {
               },
               {
                 label: 'Discord',
-                href: 'https://discordapp.com/invite/docusaurus',
+                href: 'https://discord.com/invite/JZ2btyTfzQ',
```
Comment (ardasevinc) [inline docusaurus.config.js:101]: This change also closes #313 but you were assigned to #313. Please keep the PR's atomic (one change only). I'll merge this one.

**Review by ardasevinc — APPROVED** [review]
LGTM! Thanks for contributing. Try to keep the scope of your PR to one issue next time please

File: docusaurus.config.js (hunk 1 above)
Comment (RakeshSangem) [reply docusaurus.config.js:101]: Sure, Thanks!

**Pull request merged**